app/assets/stylesheets/charts.scss

```diff
@@ -143,6 +143,13 @@
   font-size: 14px;
 }
 
+.metrics-chart {
+  width: 240px;
+  margin: 12px auto;
+  margin-top: 50px;
```

Comment:
duplicate `margin-top` with 12px and 50px values.
maybe use margin with 4 values?